app/views/hiring_staff/vacancies/application_details/show.html.haml

```diff
@@ -0,0 +1,49 @@
+- if @vacancy.published?
+  - content_for :page_title_prefix, "#{@application_details_form.errors.present? ? 'Error: ' : ''}Edit the #{t('jobs.application_details')}"
```

Comment:
This page title  logic could be moved in an helper as it is duplicated in many places, maybe a PR only on that would be nice